tests/integration/kubernetes/k8s-policy-pvc.bats

```diff
@@ -9,6 +9,7 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
+	( [ "${KATA_HYPERVISOR}" == "qemu-sev" ] || [ "${KATA_HYPERVISOR}" == "qemu-snp" ] ) && skip "https://github.com/kata-containers/kata-containers/issues/9846"
```

Comment:
Let's move the new skip after the older skip below. No need to point to the PVC issue if auto_generate was disabled.

Reply:
Updated